README.md

```diff
@@ -15,35 +15,36 @@ application to capture and report distributed traces to Splunk APM.
 This Splunk distribution comes with the following defaults:
 
 - [B3 context propagation](https://github.com/openzipkin/b3-propagation).
-- [Jaeger thrift
-  exporter](https://opentelemetry-python.readthedocs.io/en/stable/exporter/jaeger/jaeger.html)
-  configured to send spans to a locally running [SignalFx Smart
-  Agent](https://docs.signalfx.com/en/latest/apm/apm-getting-started/apm-smart-agent.html)
-  (`http://localhost:9080/v1/trace`).
-- Unlimited default limits for [configuration options](#trace-configuration) to
+- [Jaeger Thrift over HTTP
+  exporter](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/jaeger)
+  configured to send spans to a locally running Splunk OpenTelemetry Connector](https://github.com/signalfx/splunk-otel-collector)
```

Comment:
```suggestion
  configured to send spans to a locally running Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
```

Reply:
The name is "Connector" 😄 
See: https://github.com/signalfx/splunk-otel-collector
There was a big thread on this but this is the term that we should use (sic!).

Reply:
Gotcha :+1: